data/universities.js

```diff
@@ -106,15 +105,18 @@ async function updateUniversity(name, emailDomain) {
 }
 
 async function deleteUniversity(id) {
+  sharedValidation.checkArgumentLength(arguments.length, 1);
+  validation.isValidUniversityId(id);
+
   const universitiesCollection = await universities();
-  const university = await universities.findOne({ _id: id });
+  const university = await universityCollection.findOne({ _id: ObjectId(id) });
 
   if (!university) {
     throw 'No university with given id exists!';
   }
 
   const deleteUniversity = await universitiesCollection.deleteOne({
-    _id: id
+    _id: ObjectId(id)
   });
 
   if (deleteUniversity.deletedCount == 0) {
```

Comment:
@gpark31 can we revert the changes to this file? Everything was taken care of in https://github.com/dwilliamson5/CS546_WS_final_project/pull/38

Reply:
all validations as well?

Reply:
i made a different commit with just validations for items and data/item fixed and lemme add my seed